setup.cfg

```diff
@@ -14,6 +14,7 @@ classifiers =
     License :: OSI Approved :: Apache Software License
     Programming Language :: Python :: 3
 
+# code was previously (<= 0.0.23) a
 [options]
```

Comment:
?  Left over note to yourself?

Reply:
I meant to leave a comment but removed it later, will remove.